src/plugins/dashboard/public/dashboard_app/_dashboard_app_strings.ts

```diff
@@ -26,9 +26,21 @@ export const dashboardReadonlyBadge = {
 };
 
 export const dashboardManagedBadge = {
-  getTooltip: () =>
+  getText: () =>
     i18n.translate('dashboard.badge.managed.tooltip', {
-      defaultMessage: 'Elastic manages this dashboard. Clone it to make changes.',
+      defaultMessage: 'Elastic manages this dashboard.',
+    }),
+  getManagedText: () =>
+    i18n.translate('dashboard.managedContentBadge.text', {
+      defaultMessage: 'Managed',
+    }),
```

Comment:
We should remove this - in our call to `getManagedContentBadge`, we could provide the `ariaLabel` instead since this is used as our "tooltip text" (even though we aren't using it)

Reply:
I think I'm following, let me know if I'm misunderstanding, thanks!

Reply:
Ah, took a look, and not quite - we definitely don't want two calls to `getManagedContentBadge`. All I'm saying is we should provide `dashboardManagedBadge.getAriaLabel()` as the `tooltipText` in the call :)